Makefile

```diff
@@ -163,7 +185,7 @@ run-local:
 	go run ./main.go
 
 login:
-	@docker login -u "$(DOCKER_USER)" -p "$(DOCKER_PASS)"
+	echo "$(DOCKER_PASS)" | docker login -u "$(DOCKER_USER)" --password-stdin
```

Comment:
Here you change the `login` target command-line to match the `test-login` command-line, but then you don't use it above in the GitHub Actions workflow during the publish. Does `make login` not work? Do we need to bind the secrets as environment variables before calling `make login`?

Here is an example of binding secrets to environment variables: https://github.com/pravega/pravega/blob/master/.github/workflows/gradle.yml#L152